doc/reference/devices_disk.md

```diff
@@ -79,6 +79,7 @@ Key                 | Type      | Default   | Required  | Description
 `boot.priority`     | integer   | -         | no        | Boot priority for VMs (higher value boots first)
 `ceph.cluster_name` | string    | `ceph`    | no        | The cluster name of the Ceph cluster (required for Ceph or CephFS sources)
 `ceph.user_name`    | string    | `admin`   | no        | The user name of the Ceph cluster (required for Ceph or CephFS sources)
+`io.cache`          | string    | -         | no        | Override the caching mode for the device. One of `none` (default), `writeback` or `unsafe` (VM only)
```

Comment:
In https://github.com/lxc/lxd/pull/11468/files#diff-8bec2f9c6e6273e7d5ac8f05bc48859138ae35e34d5d5a93cb4d8adc58f5fa5cR29 @ru-fu uses the "Only for containers: " prefix rather than putting it in brackets at the end like it is here.

We should agree and use a consistent approach to marking config items as instance-type specific.

Reply:
I noticed we're inconsistent in where we put this information. When the description ends with a part in brackets, we can't have another set of brackets, so it should rather go in the front.
Ideally, we should have an extra column for it, but that will make the tables harder to read ... so we might want to postpone this until we use the new format for config options.

Here's my suggestion for this case (we don't want a period in the description):

```suggestion
`io.cache`          | string    | `none`         | no        | Only for VMs: Override the caching mode for the device (`none`, `writeback` or `unsafe`)
```